redfish/rules/dell_redfish_1.0.2.yml

```diff
@@ -0,0 +1,108 @@
+Traverse:
```

Comment:
Where is the specification of this file format?

Reply:
Nowhere.
I'll add the spec.

I didn't make a document for the format because it was so simple and it seemed to me that the developers could understand how to write rules without written spec.
But now it is somewhat complicated.
Anyway, it is not good to leave the original file format undocumented...